eo-runtime/src/main/java/org/eolang/PhDefault.java

```diff
@@ -284,6 +284,16 @@ public String locator() {
         return "?";
     }
 
+    /**
+     * Clean up resources.
+     * This includes call of {@link ThreadLocal#remove()} method to prevent
+     * memory leaks.
+     */
+    public static void cleanUp() {
```

Comment:
@mximp I don't see anyone calling this method. I think, `remove()` is supposed to be called right after we use `TERMS`, not in a separate method. But I'm not sure...

Reply:
@yegor256 You are right - the method is not called by anyone. The thing is I cannot define the moment "right after TERM is used". It's static and holds the value which is required throughout the program execution. So I see correct place for `remove` to be called before program exits. But that doesn't make sense.
So in this case I just created `fake` method according to SonarQube expectations.